cmd/celestia-appd/cmd/root.go

```diff
@@ -100,9 +99,6 @@ func NewRootCmd() *cobra.Command {
 
 // initRootCommand performs a bunch of side-effects on the root command.
 func initRootCommand(rootCommand *cobra.Command, encodingConfig encoding.Config) {
-	config := sdk.GetConfig()
```

Comment:
Was this line meant to be deleted? Asking in case it was not intentional

Reply:
Yep it meant to be deleted. It doesn't need to be invoked here because it's already invoked via `init()`.

Reply:
>it's already invoked via init().

By `init()`, do you mean the following function from`app/sdk_config.go`?
```go
func init() {
	setCosmosSDKConfig()
}
```
If yes, then it seems it has no usage in the codebase (I looked up for its usage in your branch, and nothing showed up). Am I missing something?

Reply:
`init` is automatically invoked. https://go.dev/doc/effective_go#init